src/main/java/org/spongepowered/common/mixin/core/entity/EntityLivingBaseMixin.java

```diff
@@ -588,6 +590,14 @@ public boolean attackEntityFrom(final DamageSource source, final float amount) {
         }
     }
 
+    @Redirect(method = {"<init>", "readEntityFromNBT"}, at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/EntityLivingBase;setHealth(F)V"))
```

Comment:
Javadoc why this is being done, link to the issue if need be. on the surface it doesn't look like it's changing anything except using a more direct approach to setting health (which honestly, might affect some mods).